WhaleSpotting/Models/DbModels/SightingDbModel.cs

```diff
@@ -21,7 +25,9 @@ public class SightingDbModel
         public string OrcaPod { get; set; }
         public bool Confirmed { get; set; }
 
-        // TODO - Add UserDbModel
-        //public UserDbModel User {get; set; }
-    }
+        public SightingDbModel ()
+        {
+
+        }
```

Comment:
Can we put the TODO back and remove this constructor

Reply:
:)